From 0c4395f039b4309133462c1eae027bb733aebdcc Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Thu, 10 Feb 2022 09:51:59 -0500
Subject: [PATCH] Avoid test suite failure when only compression-deflate is
 enabled.

compressed-data-algo-3.gpg contains a bzip2 compression layer.  This
test was behind a compression-deflate feature flag, but it's possible
to enable compression-deflate without compression-bzip2, which would
cause this test to fail.

Closes: #823

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
---
 src/serialize.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/serialize.rs b/src/serialize.rs
index 6656e439..5de5e506 100644
--- a/src/serialize.rs
+++ b/src/serialize.rs
@@ -3092,7 +3092,7 @@ mod test {
 
     }
 
-    #[cfg(feature = "compression-deflate")]
+    #[cfg(feature = "compression")]
     #[test]
     fn serialize_test_2() {
         // Given a packet in serialized form:
-- 
2.34.1

